File: Python/1176.py
limite = 61
pos = 0
vet = [0 for i in range(limite)]
vet[0], vet[1] = 0, 1
for i in range(2, limite):
    vet[i] = vet[i - 1] + vet[i - 2]
qntd = int(input())
for i in range(qntd):
    pos = int(input())
    print("Fib({}) = {}".format(pos, vet[pos]))

File: C99/1176.c
#include <stdio.h>
#define limite 61
int main() {
	// long long int -> Declara variáveis inteiras de 64 bits
    long long int vet[limite], qntd;
    int i, posicao;
    vet[0] = 0;
    vet[1] = 1;
    for(i = 2; i < limite; i++)
    	vet[i] = vet[i - 1] + vet[i - 2];
    scanf("%lld", &qntd);
    for(i = 0; i < qntd; i++) {
    	scanf("%d", &posicao);
    	printf("Fib(%d) = %lld\n", posicao, vet[posicao]);
    }
    return 0;
}

